brd-ios/breadwallet/src/Redesign/Core/Views/SubviewConfigurable.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  SubviewConfigurable.swift
+//  
+//
+//  Created by Rok Cresnik on 06/09/2021.
+//
+
+import UIKit
+
+/// Used for configuring multiple subview elements from the container class
+protocol SubviewConfigurable {
+    associatedtype Views
+    func configureSubviews(configure: (Views) -> Void)
+}
+
+//        // e.g.
```

Comment:
👀

Reply:
this is litterally a "how to use" comment :)